packages/uni_ui/lib/modal/widgets/info_row.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+import 'package:phosphor_flutter/phosphor_flutter.dart';
+
+class ModalInfoRow extends StatelessWidget {
+  const ModalInfoRow(
+      {super.key,
+      required this.title,
+      required this.description,
+      required this.icon});
+
+  final String title;
+  final String description;
+  final PhosphorIconData icon;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+        padding: const EdgeInsets.all(3.0),
+        decoration: BoxDecoration(
+            border: Border(
+                top: BorderSide(
+                    color: Theme.of(context).dividerColor, width: 1))),
+        child: Row(
+          children: [
+            Expanded(
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  Text(
+                    title,
+                    style: Theme.of(context).textTheme.bodyMedium,
+                  ),
+                  Text(description,
+                      style: Theme.of(context).textTheme.bodySmall!.copyWith(
+                          color: Theme.of(context).secondaryHeaderColor)),
+                ],
+              ),
+            ),
+            PhosphorIcon(icon as IconData),
```

Comment:
Don't forget to add a `IconButton` with an action passed by the father.
Also, could we already define icon as `IconData` to avoid `as IconData` ?

Reply:
yes ahah. For example, the pin redirects to map, the telephone to callers app.
I will send this feedback to designers team as we can see is not intuitively.

Reply:
Okay, in the meanwhile I'll put the IconButton.